prime-router/src/test/kotlin/fhirengine/engine/FhirRouterTests.kt

```diff
@@ -789,6 +810,100 @@ class FhirRouterTests {
         }
     }
 
+    @Test
+    fun `success - everything passes, send original`() {
```

Comment:
We did not change anything with the route function, why does this need a new test? What value does this add?